config_defaults.yml

```diff
@@ -41,6 +41,8 @@ appveyor.yml:
       RUBY_VER: 21-x64
   test_script:
   - "bundle exec rake spec SPEC_OPTS='--format documentation'"
+MAINTAINERS.md:
+  maintainers: "Puppet Forge Modules Team <forge-modules@puppet.com>"
```

Comment:
This is a default in case a module has not specified one.  As this is across all Puppet modules, the only common entity for that is the Forge Modules team.  Ideally each module would override this.